### PR TITLE
doc: Documentation tweaks ALA-607

### PR DESCRIPTION
**Pull request by nicklem**
Documentation tweaks

### TODO

- [x] ❗ Merge **after** https://github.com/codacy/docs/pull/1851

### CODE REVIEW

**Review by machadoit — DISMISSED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 # Codacy Visual Studio Code Extension
 
-The Codacy extension for Visual Studio Code helps you review and manage issues found by Codacy by highlighting problematic code patterns, notifying you whether a pull request is up to standards, and displaying code quality metrics directly within Visual Studio Code.
+The Codacy extension for Visual Studio Code helps you review and manage issues found by Codacy by notifying you whether a pull request is up to standards, highlighting problematic code patterns, and displaying code quality metrics directly within Visual Studio Code.
```
Comment (claudiacarpinteiro) [inline README.md:3]: Small tweaks.

```suggestion
The Codacy extension for Visual Studio Code helps you review and manage the issues found by Codacy directly within Visual Studio Code. It notifies you whether a pull request is up to standards by highlighting problematic code patterns and displaying code quality metrics.
```

**Review by claudiacarpinteiro — DISMISSED** [review]
Looks good to go! 🚀 
Just added a couple of tweaks for your evaluation.

**Review by claudiacarpinteiro — APPROVED** [review]
LGTM! 🚀

**Pull request merged**